optd-persistent/src/cost_model/interface.rs

```diff
@@ -40,7 +46,10 @@ pub enum ConstraintType {
 }
 
 /// TODO: documentation
+#[derive(Copy, Clone, Debug, PartialEq)]
 pub enum StatType {
+    /// A combination of multiple statistics, e.g. most common values, distribution.
+    Comb,
```

Comment:
I don't think there is a stat type called `Comb`, since multi-attributes (comb stats) can have `NotNullCount`, `Cardinality`, etc. Also, single attribute statistic type can be `MostCommonValues`, `Distribution`...
I would suggest changing it to MostCommonValues, Distribution, Histogram...